packages/ai-jsx/src/lib/openai.tsx

```diff
@@ -370,6 +442,14 @@ export async function* OpenAIChatModel(
             },
           };
         case 'functionResponse':
+          if (message.element.props.id && !unmatchedIds.has(message.element.props.id)) {
```

Comment:
Does OAI throw an error for unmatched/incorrectly matched function responses? If not, do we really need to be so meticulous in removing them?

Reply:
They do :)